### DISCUSSION TITLE
Alpha Optimization Test Almost Complete

### DISCUSSION THREAD

**Discussion by Mrbt0907** [CosmicHikari/Weather2-Remastered#26] [Announcements]
Great news! I fixed 90% of the features that broke during the netcode rework (Which is still under works). A lot of code was changed, removed, and should in theory be faster. All I have to do now is get all the new configs to work and test things out with you guys.

However, I did do some testing on my own with the build I have now, and based on my testing, the lag seemed to have gotten worse. Normal weather2 had 1100 microseconds of lag with an ef5, and the ef2 (Which is now the same size as the previous ef5) took 2600~ microseconds. That is not good, but was not a full test. I'll let you guys do that soon. Hopefully I can fix my mistake if it is real.

What I am proud of however is the big improvements with the configurations and features of the storms compared to the current weather2 mod. Most if not all old configurations will not work for this version, so you will have to reconfigure this mod when it is ready. There should be a good amount of new configurations to play around with, and 99% of them have definitions!

For now, I will leave you with a picture showing the new EF5 damage vs the old EF5 damage. Keep giving me suggestions and bugs and I will get to it as soon as I can. I hope you guys have a great day! 
![2022-02-01_18 03 14](https://user-images.githubusercontent.com/50820746/152086130-e2c8ee49-7d73-47d6-806c-07abec92a0c3.png)

**Comment by OttovonBismarckderErste** [comment]
Happy to hear that :)

**Comment by zad112** [comment]
sweet im excited to check it out amazing work. 
also while i'm not the best at coding (can do textures n stuff) I'm also extremely well versed in meteorology so if you have any questions or need any help lmk. I have always loved this mod and seeing a new version of it come alive is amazing. Not just a port but a whole new world.

**Reply by CosmicHikari** [reply]
I can do a little programming, I'm well versed with making stuff in mcreator and editing it in eclipse so if I can be of any use towards this project consider me in.

**Comment by CosmicHikari** [comment]
Just saw this - Looks GREAT. Much better than the original EF5 damage.

**Comment by zad112** [comment]
Looks great can't wait to see what this eventually becomes. It has the potential to become MUCH bigger and could even become the de facto minecraft weather mod givin some time and mojang doesn't go implementing localized weather (doubt it). 
also can't code for crap but if you have texture/modeling stuff still needed lmk thats my jam
